build/hostlogswindows/installer/startupprobe/startupprobe.cpp

```diff
@@ -0,0 +1,76 @@
+#ifndef UNICODE
+#define UNICODE
+#endif
+
+#ifndef _UNICODE
+#define _UNICODE
+#endif
+
+#include <Windows.h>
+#include <tlhelp32.h>
+#include <iostream>
+
+#define SUCCESS 0x00000000
+#define NO_MONAGENT_LAUNCHER_PROCES 0x00000001
+#define UNEXPECTED_ERROR 0xFFFFFFFF
+
+/*
+* Function to check if a process is running based on its full path
+*/ 
+bool IsProcessRunningWithSpecifcPath(const wchar_t* executableFullPath) {

```

Comment:
Is there any way we could share this function with the liveness probe rather than maintaining 2 copies of the same thing?

Reply:
+1 to a thread Alberto made before this. Closing this as a duplication.